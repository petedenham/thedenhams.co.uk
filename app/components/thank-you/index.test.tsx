import ThankYou from ".";
import { render, screen } from "@testing-library/react";

describe("ThankkYou", () => {
  it("matches snapshot", () => {
    const { container } = render(<ThankYou />);

    expect(container).toMatchSnapshot();
  });
});
