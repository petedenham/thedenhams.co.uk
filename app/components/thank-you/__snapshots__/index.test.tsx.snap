// Vitest Snapshot v1

exports[`ThankkYou > matches snapshot 1`] = `
<div>
  <h1
    class="text-xl mb-5 font-bold text-center lg:text-left"
  >
    Thank you confirming if you can attend
  </h1>
  <p>
    We can't wait to see you on the dance floor!
  </p>
</div>
`;
